oauth2/session_custom_claims_test.go

```diff
@@ -0,0 +1,137 @@
+package oauth2_test
+
+import (
+	"fmt"
+
+	"testing"
+
+	"github.com/ory/fosite/handler/openid"
+	"github.com/ory/fosite/token/jwt"
+
+	"github.com/ory/hydra/oauth2"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func createSessionWithCustomClaims(subject string, extra map[string]interface{}, issuer string) oauth2.Session {
+	session := &oauth2.Session{
+		DefaultSession: &openid.DefaultSession{
+			Claims: &jwt.IDTokenClaims{
+				Subject: subject,
+				Issuer:  issuer,
+			},
+			Headers: new(jwt.Headers),
+			Subject: subject,
+		},
+		Extra: extra,
+	}
+
+	return *session
+}
+
+func TestCustomClaimsInSession(t *testing.T) {
```

Comment:
Here you would use
```suggestion
func TestCustomClaimsInSession(t *testing.T) {
	c := internal.NewConfigurationWithDefaults()
```
to then pass it down to the session.